packages/ui/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "ui",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@primer/components": "^27.0.0",
```

Comment:
I thought we talked about using Stacks UI Components last Friday. Did we change our minds at some point?

Reply:
@314159265359879 & I discussed using Primer instead as it matches our values/purpose more. [link](https://discord.com/channels/621759717756370964/824629071387426838/834402216978546719). though if you'd prefer to use the Stacks UI components we can vote on it 😄

Reply:
@pxydn I have no strong preference, although Primer design system looks a bit dull to me, but maybe it's because I'm too used to Github. It certainly means less work since the library is a bit more mature than Stacks one.

Reply:
I haven't looked at the Stacks UI component library yet. Have you guys? Does it really provide additional value over other popular choices that are more widely supported?

Reply:
```suggestion
```
We could just delete this, merge if all else is approved, and come back to the component question later.

Reply:
It provides a more unique look&feel. If the decision is going to be based on value and popularity, we should be using Tailwind UI or Material UI.

Reply:
+ 1 for Tailwind over Material UI (we could also have a Tailwind based L3 UI themed component set we can make). But I will also look at Stacks UI. @marbiano

Reply:
Seems to me like the UI library should not be part of this PR.

Reply:
@JMKJR @marbiano i'm removing the UI components off here for now. then we can get the go ahead's and merge into main.